src/app/src/androidTest/java/com/uphold/uphold_android_sdk/test/integration/model/UserTest.java

```diff
@@ -129,6 +130,56 @@ public void createContactShouldReturnTheContact() throws Exception {
     }
 
     @Test
+    public void getAccountByIdShouldReturnTheAccountWithId() throws Exception {
+        String responseString = "{ \"id\": \"FOOBAR\" }";
```

Comment:
This test should be done with a full account `json` to test that the `json` is correctly matched to the model.